pkgs/os-specific/linux/numworks-udev-rules/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "numworks-udev-rules";
+  version = "unstable-2020-08-31";
+
+  udevRules = fetchurl {
+    # let's pin the latest commit in the repo which touched the udev rules file
+    url = "https://workshop.numworks.com/files/drivers/linux/50-numworks-calculator.rules";
+    sha256 = "1k3vl4q8pivk1dm6inb56ij4shjs17sn8lcnq8bxnx4j5d1mx2f7";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    install -Dm 644 "${udevRules}" "$out/lib/udev/rules.d/50-numworks-calculator.rules"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Udev rules for Numworks calculators";
+    homepage = "https://numworks.com";
+    license = licenses.gpl3;
```

Comment:
`gpl3` is deprecated, use `gpl3Only` or `gpl3Plus` (if the software is licensed under the GPL3 or at the user's option a later version)

Reply:
Wrote that as default. I've been contacting numworks to ask them under which license this file is protected. Waiting for there answer...

Reply:
Got an answer in less than 24h. Really nice customer service ! They don't really license this file for know but they said I can safely put it under mit license.